translations/texts/objects/apex/apexconsolekeyboard/apexconsolekeyboard.object.json

```
@@ -56,7 +56,7 @@
     },
     "Texts": {
       "Eng": "Hylotl lettering flows like water. Apex lettering flows like stone.",
-      "Rus": ""
+      "Rus": "Надпись Хилотл течет как вода. Апексная надпись течет как камень."
```

```suggestion
      "Rus": "Письменность хилотлов льётся как вода. Письменность апексов же, словно камень."
```
Что это вообще было?